deploy/charts/venafi-kubernetes-agent/templates/deployment.yaml

```diff
@@ -32,8 +32,24 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          {{- if or .Values.http_proxy .Values.https_proxy .Values.no_proxy }}
           env:
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POD_UID
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.uid
+          - name: POD_NODE
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          {{- if or .Values.http_proxy .Values.https_proxy .Values.no_proxy }}
```

Comment:
I think this if statement can be removed, because its original purpose was to omit the `env` stanza if no proxy env vars were needed. Now the `env` stanza is always included. I'll remove it.